pkg/cmd/join/exec.go

```diff
@@ -49,15 +54,27 @@ func (o *Options) complete(cmd *cobra.Command, args []string) (err error) {
 	// convert mode string to lower
 	o.mode = strings.ToLower(o.mode)
 
+	// In default deploy mode, klusterlet is deployed in operator namespace "open-cluster-management", the agents(such as registration-agent) are
+	// deployed in agent namespace "open-cluster-management-agent".
+	// In hosted deploy mode, klusterlet and agents are both deployed in agent namespace(the agent namespace is on the management cluster, and share
+	// name with klusterlet); the klusterletNamespace is still on the managed cluster, the name is "open-cluster-management-<klusterlet-name>".
+	klusterletName := "open-cluster-management"
```

Comment:
this is a little confused, the default klusterlet name is "open-cluster-management"?